resolwe_bio/processes/chip_seq/macs2.yml

```diff
@@ -536,6 +559,9 @@
 
         # Single-end reads
         if [ $(samtools view -c -f 1 {{ alignment.bam.file }}) -eq 0 ]; then
+          {% if settings.format == "BAMPE" %}
```

Comment:
Couldn't we decide between BAM and BAMPE automatically?

Reply:
This is up for discussion. It is indeed possible, but it would significantly change the process. First, only the reads that are proper pairs would be considered and insert size would be use to build fragment pileup. Based on a few posts this option seems to be a preference rather than a standard choice and by choosing it manually makes the process more universal.

Reply:
Ok, than it is best to leave it as is, being user configurable.